pom.xml

```diff
@@ -755,7 +756,12 @@
       </dependency>
       <dependency>
         <groupId>org.apache.logging.log4j</groupId>
-        <artifactId>log4j-slf4j-impl</artifactId>
+        <artifactId>${log4j.slf4j.impl}</artifactId>
+        <version>${log4j.version}</version>
+      </dependency>
+      <dependency>
```

Comment:
hmm... Why are both `log4j.slf4j.impl` and `log4j-slf4j2-impl` configured here?

Reply:
The purpose of `dependencyManagement` in the root pom.xml is to specify the version of the dependencies used in the entire project, regardless of whether you have used it or not. My original intention was to define it uniformly here, so I don't need to write it separately in different profiles. If you want to separate it into two parts and put them in the `dependencyManagement` of the `spark3.4` and `spark3.5` profiles, that's also possible. I can get this done.

Reply:
I think `<artifactId>${log4j.slf4j.impl}</artifactId>` is the original code, not the latest code. I cannot find  `<artifactId>${log4j.slf4j.impl}</artifactId>` in my code anymore.